src/plugins/StoreyViewsPlugin/StoreyViewsPlugin.js

```diff
@@ -696,9 +697,10 @@ class StoreyViewsPlugin extends Plugin {
      * @param {Number[]} worldPos 3D World-space position.
      * @returns {String} ID of the storey containing the position, or null if the position falls outside all the storeys.
      */
-    getStoreyInVerticalRange(worldPos) {
-        for (let storeyId in this.storeys) {
-            const storey = this.storeys[storeyId];
+    getStoreyInVerticalRange(worldPos, modelId = "") {
```

Comment:
Let's make modelId null by default. 
(Even though JS so weirdly equates "" to false!)